okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -174,6 +187,27 @@ private boolean resolveRedirects(ProxyContext pc,
     return true;
   }
 
+  private boolean checkTokenCache(ProxyContext pc, HttpServerRequest req, RoutingEntry re,
+      ModuleInstance mi) {
+    boolean skipAuth = false;
```

Comment:
This variable is not needed. We can directly return true or false.
There should be javadoc explaining the return value.

Reply:
done